docker/docker.mk

```diff
@@ -121,6 +121,14 @@ else
 	$(DOCKER) buildx build --build-arg PKG_FILES=sentry --platform $(DOCKER_IMAGE_PLATFORM) -f $(DOCKERFILE_DIR)/$(DOCKERFILE) $(BIN_PATH) -t $(DAPR_SENTRY_DOCKER_IMAGE_TAG)-$(TARGET_OS)-$(TARGET_ARCH) --push
 endif
 
+# push docker image for e2e tests
+docker-push-e2e:
```

Comment:
Why do we need a separate target for this? Isn't docker-push above sufficient?

Reply:
`docker-push` depends on `docker-build` :(

Reply:
I would rather create a special env variable that will skip the build and keep the docker-push target above only. There is more logic to support multi arch, for example, that this one does not replicate.